### ISSUE TITLE
deno - publish fixed imports

### ISSUE THREAD

**Issue by DhruvDh** [NSSTC/sim-ecs#69]
Imports for deno were fixed in https://github.com/NSSTC/sim-ecs/commit/ac58bd7761dab56d69b4e0b90facbfe7b206fb41, however it is not possible to import this fixed version from deno.

Can a new (minor) release be pushed to deno or npm?

**Comment by minecrawler** [comment]
Hi @DhruvDh ,

Thank you for expressing your interest in this fix! I've been working on a few features for the next update, but they aren't ready just yet. However, I'll go ahead and release the bug fix early, so you can go about your business :)

**Comment by DhruvDh** [comment]
Thanks! I think I'll end up using this client-side, deno is just very easy to start experimenting with.
I think publishing soon is a good idea as people who find and try this from deno third party marketplace won't end up thinking it is broken.

**Comment by minecrawler** [comment]
@DhruvDh I published v0.6.3. Testing it from the Deno playground, at least the import works fine, now. Please check if this fixes your problem!

```
import * as simEcs from "https://deno.land/x/sim_ecs@v0.6.3/src/index.ts";
```

Happy coding :)
